identify/README.md

```diff
@@ -133,3 +134,14 @@ clients only support initiating requests while some servers (only) support
 responding to requests. To prevent clients from initiating requests to other
 clients, which given them being clients they fail to respond, clients should not
 advertise `foo` in their `protocols` list.
+
+### signedPeerRecord
+
+This is a serialized [SignedEnvelope][envelope-rfc] containing a [PeerRecord][peer-record-rfc],
+signed by the sending node. It contains the same addresses as the `listenAddrs` field, but in a form that lets us share authenticated addrs with other peers.
+
+This field was introduced in a backwards compatible manner (meaning that it is sent along with the `listenAddrs` field), therefore, it is optional and may be omitted by older implementations. If the `signedPeerRecord` is present and the implementation supports SignedEnvelope, it MUST use the data contained within it and ignore duplicated fields present in the main identify message
```

Comment:
Ah sorry - I [meant](https://github.com/libp2p/specs/pull/630#discussion_r1753553493) following an earlier version of the spec was fine.

Suggested edit:

```suggestion
This field was introduced in a backwards compatible manner (meaning that it is sent along with the `listenAddrs` field), therefore it is optional. If the `signedPeerRecord` is present the implementation MUST use the data contained within it and ignore duplicated fields present in the main identify message
```

- The field is optional so it feels redundant to explicitly call out that it can be omitted
- In this version of the spec acting on the field if it is present is not optional, that's the point of the `MUST`
- Don't forget to rev the `Latest Revision` field in the header..